### PR TITLE
docs: update the prerequisites page

### PR DESCRIPTION
**Pull request by anupriya0703**
Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: quickstart/prerequisites.md [hunk 1]
```diff
@@ -8,29 +8,272 @@ All worker nodes must satisfy the following requirements:
...
+  apiGroup: rbac.authorization.k8s.io
+```
+
+{% endtab %}
+
+   
+
+## Minimum worker node count
+
+  The minimum supported worker node count is three nodes. When using the synchronous replication feature (N+1 mirroring), the number of worker nodes to which Mayastor is deployed should be no less than the desired replication factor.
```
Comment (GlennBullingham) [inline quickstart/prerequisites.md]: I think that should be "(N-way mirroring)"

**Review by GlennBullingham — CHANGES_REQUESTED** [review]

File: quickstart/prerequisites.md (hunk 1 above)
Comment (anupriya0703) [reply quickstart/prerequisites.md]: Updated it.

File: quickstart/prerequisites.md [hunk 2]
```diff
@@ -8,29 +8,272 @@ All worker nodes must satisfy the following requirements:
...
+
+## Minimum worker node count
+
+  The minimum supported worker node count is three nodes. When using the synchronous replication feature (N-way mirroring), the number of worker nodes to which Mayastor is deployed should be no less than the desired replication factor.
+
+
+## Transport protocols
+
+    Mayastor supports the export and mounting of volumes over NVMe-oF TCP only. Worker node(s) on which a volume may be scheduled (to be mounted) must have the requisite initiator support installed and configured.
+    In order to reliably mount application volumes over NVMe-oF TCP, a worker node's kernel version must be 5.13 or later and the nvme-tcp kernel module must be loaded.
```
Comment (avishnu) [inline quickstart/prerequisites.md]: Mayastor volumes, not application volumes?

File: quickstart/prerequisites.md [hunk 3]
```diff
@@ -8,29 +8,272 @@ All worker nodes must satisfy the following requirements:
...
   * ext4 and optionally xfs
+* * [Helm](https://helm.sh/docs/intro/install/) version must be v3.7 or later.
 
-## **Mayastor pods (Data Plane/Engine)**
-Each worker node which will host an instance of a Mayastor pod must have the following resources _free and available_ for _exclusive_ use by that pod:
 
-* Two CPU cores
-* 4GiB RAM
-* **HugePage support**
-  * A minimum of **2GiB of** **2MiB-sized** pages
```
Comment (avishnu) [inline quickstart/prerequisites.md:18]: Is the above not needed?

Comment (anupriya0703) [reply quickstart/prerequisites.md:18]: The above has been added under 'recommended resource requirements'. (from line 31)

Comment (avishnu) [reply quickstart/prerequisites.md:18]: Those seem to be the resource requests and limits per Mayastor component in K8s.
Above is a generic requirement for the node as a whole.

Comment (anupriya0703) [reply quickstart/prerequisites.md:18]: Okay. I'll add it back.

**Review by avishnu — APPROVED** [review]
lgtm

**Review by GlennBullingham — APPROVED** [review]

**Pull request merged**